frontend/taipy-gui/src/components/Taipy/Field.tsx

```diff
@@ -42,6 +44,13 @@ const Field = (props: TaipyFieldProps) => {
 
     const mode = typeof props.mode === "string" ? props.mode.toLowerCase() : undefined;
 
+    const mathJaxConfig = {
```

Comment:
It is better to declare this outside the component, otherwise on each render it will be a new object.
Before line 38